pkgs/servers/xmpp/biboumi/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, cmake, libuuid, expat, sqlite, git, libidn,
+  libiconv, botan2, systemd, pkgconfig, udns } :
+
+stdenv.mkDerivation rec {
+  name = "biboumi-${version}";
+  version = "6.1";
+
+  src = fetchurl {
+    url = "https://git.louiz.org/biboumi/snapshot/biboumi-${version}.tar.xz";
+    sha256 = "1la1n502v2wyfm0vl8v4m0hbigkkjchi21446n9mb203fz1cvr77";
+  };
+
+  buildInputs = [ cmake libuuid expat sqlite git libiconv libidn botan2
+    systemd pkgconfig udns ];
```

Comment:
How is git used during the build?

Reply:
Apparently a test suite is fetched. I don't like this, too.

Reply:
It will also fail to build in sandbox builds, which is what we use on hydra. So this behavior has to be prevented.

Reply:
almost done :)

Reply:
It seems not to download anything during build. So this is fine.

Reply:
Fine, but I already had decided to patch it away so I can enable tests. :-)